### ISSUE TITLE
Dynamic import of language files results in runtime errors when bundled for browser use

### ISSUE THREAD

**Issue by harrygr** [dettier/sculp#5]
Due to the way Sculp requires its language files in lib/i18n/lang.js: 

```javascript
function getMessages(lang) {
  if (lang == null) return {};
  return require('./languages/' + lang + '.js');
}
```

compilers such as browserify cannot know which files to import when compiling. This results in an error like so:

> Cannot find module './languages/en.js'

**Comment by dettier** [comment]
Thanks, I will look into it.

**Comment by dettier** [comment]
This should be fixed now in 3.0.0. Thanks for submitting!
